### ISSUE TITLE
Unnecessary scanning of Microsoft Flight Simulator (migrated to second drive)

### ISSUE THREAD

**Issue by needsMoreTurbo** [mtkennerly/ludusavi#243]
### Ludusavi version

v0.20.0

### Operating system

Windows

### Installation method

Standalone

### Description

I have Microsoft Flight Simulator installed on a second drive and every time Ludusavi scans, it gets stuck scanning every file within the installation directory (~200 GB). It would probably finish if I left it alone for a couple hours, but typically Windows Defender kicks in and starts bogging it down.

A snippet from the Rust log is attached, but essentially there is a line for every file in the installation until the log file reaches 10MB

I resolved this by ignoring the corresponding path, but is there a way to prevent this all together? Adding "Microsoft Flight Simulator (2020)" to the ignored games in the config file did not resolve the issue.

```
    ignoredPaths:
      - "D:/WpSystem/S-1-5-21-818977104-3589193027-2243567035-1001/"
```

I am using v0.20.0 on Win11, installed using winget.

### Logs

```
[2023-07-17T00:59:33.133Z] DEBUG [ludusavi::scan] [Microsoft Flight Simulator (2020)] found: D:/WpSystem/S-1-5-21-818977104-3589193027-2243567035-1001/AppData/Local/Packages/Microsoft.FlightSimulator_8wekyb3d8bbwe/LocalCache/Packages/Official/OneStore/asobo-flight-tutorials-newbasic/Missions/Asobo/Tutorials/Introduction/BasicControlsV2/Sound/fr-FR/BasicControls_Dlg_PlanePitchUp2_MALE.wav
[2023-07-17T00:59:33.138Z] DEBUG [ludusavi::scan] [Microsoft Flight Simulator (2020)] found: D:/WpSystem/S-1-5-21-818977104-3589193027-2243567035-1001/AppData/Local/Packages/Microsoft.FlightSimulator_8wekyb3d8bbwe/LocalCache/Packages/Official/OneStore/asobo-flight-tutorials-newbasic/Missions/Asobo/Tutorials/Introduction/BasicControlsV2/Sound/fr-FR/BasicControls_Dlg_PlanePitchUp3.wav
[2023-07-17T00:59:33.142Z] DEBUG [ludusavi::scan] [Microsoft Flight Simulator (2020)] found: D:/WpSystem/S-1-5-21-818977104-3589193027-2243567035-1001/AppData/Local/Packages/Microsoft.FlightSimulator_8wekyb3d8bbwe/LocalCache/Packages/Official/OneStore/asobo-flight-tutorials-newbasic/Missions/Asobo/Tutorials/Introduction/BasicControlsV2/Sound/fr-FR/BasicControls_Dlg_PlanePitchUp3_MALE.wav

```

**Comment by mtkennerly** [comment]
Hi! I don't see that WpSystem path on PCGamingWiki or in the manifest. Do you have a custom entry for this game?

**Comment by needsMoreTurbo** [comment]
I believe you're correct. I looked back at my installation and I believe the chronology is:

1. Originally installed the game on my boot drive through Xbox Game Pass
2. Launched MSFS, which downloaded a very large amount of data (~200 GB)
3. Migrated the installation to my secondary drive once I realized the full installation size

I've done this same process before with other XGP games, but it seems MSFS is unique and the installation files are split. 

- The XGP/Windows Store installation, which is only 2GB, is located on 
```
"D:\xboxGamePass\Microsoft Flight Simulator"
```
- Whereas the actual 200GB game installation and content is located on 
```
"D:\WpSystem\S-1-5-21-818977104-3589193027-2243567035-1001\AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalCache"
```

Regardless of the reason, it seems like my installation should fall under the "custom" category and this can be closed
